hub_module/modules/text/semantic_model/ernie_skep_sentiment_analysis/README.md

```diff
@@ -0,0 +1,177 @@
+## 概述
+
+近日，百度正式发布情感预训练模型SKEP（Sentiment Knowledge Enhanced Pre-training for Sentiment Analysis）。SKEP利用情感知识增强预训练模型， 在14项中英情感分析典型任务上全面超越SOTA，此工作已经被ACL 2020录用。
+
+SKEP（Sentiment Knowledge Enhanced Pre-training for Sentiment Analysis）是百度研究团队提出的基于情感知识增强的情感预训练算法，此算法采用无监督方法自动挖掘情感知识，然后利用情感知识构建预训练目标，从而让机器学会理解情感语义，在14项中英情感分析典型任务上全面超越SOTA。SKEP为各类情感分析任务提供统一且强大的情感语义表示。ernie_skep_sentiment_analysis Module可用于句子级情感分析任务预测。其在预训练时使用ERNIE 1.0 large预训练参数作为其网络参数初始化继续预训练。同时，该Module支持完成句子级情感分析任务迁移学习Fine-tune。
+
+<p align="center">
+<img src="https://bj.bcebos.com/paddlehub/model/nlp/skep.png" hspace='10'/> <br />
+</p>
+
+更多详情参考ACL 2020论文[SKEP: Sentiment Knowledge Enhanced Pre-training for Sentiment Analysis](https://arxiv.org/abs/2005.05635)
+
+## 命令行预测
+
+```shell
+$ hub run ernie_skep_sentiment_analysis --input_text='虽然小明很努力，但是他还是没有考100分'
+```
+
+## API
+
+```python
+def classify_sentiment(texts=[], use_gpu=False)
```

Comment:
classify_sentiment ->
sentiment_classify(xxx)

Reply:
I think classify is not a mainstream words used in interface design
How about change classify_sentiment to sentiment_predict(xxx)

Reply:
Done